### ISSUE TITLE
Possibly wrongly implemented Sound panning on 5.1+ surround setup on LWJGL (asking for test)

### ISSUE THREAD

**Issue by curioustorvald** [libgdx/libgdx#5225]
LWJGL Backend: It seems when played on surround speaker setup, Sound panning sends audio to rear speakers instead of fronts. I was able to correct this behaviour by modifying

```
FloatBuffer orientation = (FloatBuffer)BufferUtils.createFloatBuffer(6)
                                                          .put(new float[] {0.0f, 0.0f, 1.0f, 0.0f, -1.0f, 0.0f}).flip();     
```
on ```com.badlogic.gdx.backends.lwjgl.audio.OpenALAudio``` (I flipped signs).

However, my real question is: **is this behaviour driver/soundcard-dependent? Is it just me? Can you recreate the same issue? (audio plays on rear speakers when panning is centred)** I was about to make a pull request, but I couldn't test it on the various environment.

I'm including the test code. (It's written in Kotlin) You will need to put your own MONO sample as ```assets/loopey.wav```

```
package net.torvald.terrarum.audio.surroundpanner

import com.badlogic.gdx.Game
import com.badlogic.gdx.Gdx
import com.badlogic.gdx.audio.Sound
import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.badlogic.gdx.files.FileHandle
import java.awt.BorderLayout
import javax.swing.*

/**
 * Created by minjaesong on 2018-05-18.
 */
class SurroundPannerTest : JFrame() {

    val mixerPanel = JPanel(BorderLayout()) // LR slider
    
    val mixerPanSlider = JSlider(JSlider.HORIZONTAL, -32768, 32767, 0)

    init {
        val sliderPanel = JPanel(); sliderPanel.add(mixerPanSlider)
        mixerPanel.add(sliderPanel, BorderLayout.CENTER)

        this.add(mixerPanel, BorderLayout.CENTER)
        this.defaultCloseOperation = EXIT_ON_CLOSE
        this.isVisible = true
        this.setSize(400, 600)
    }

}

class AudioPlayerSlave : Game() {

    lateinit var audioSample: FileHandle
    lateinit var gdxSound: Sound
    var soundID = 0L
    lateinit var surroundPanner: SurroundPannerTest


    override fun create() {
        audioSample = Gdx.files.internal("assets/loopey.wav")
        gdxSound = Gdx.audio.newSound(audioSample)
        surroundPanner = SurroundPannerTest()
        soundID = gdxSound.loop()
    }

    override fun render() {
        gdxSound.setPan(soundID, surroundPanner.mixerPanSlider.value.toFloat() / 32768f, 1f)
    }

    override fun dispose() {
        super.dispose()
        gdxSound.dispose()
    }

}

fun main(args: Array<String>) {
    val appConfig = LwjglApplicationConfiguration()
    appConfig.vSyncEnabled = false
    appConfig.resizable = true
    appConfig.width = 256
    appConfig.height = 256
    appConfig.backgroundFPS = 20
    appConfig.foregroundFPS = 20

    LwjglApplication(AudioPlayerSlave(), appConfig)
}
```

TL;DR — Mono pans to back speakers as opposed to the front. Is it just specific to certain device or global issue?

**Comment by crykn** [comment]
I don't have a surround speak setup at hand, but testing it on all the audio devices I have here I couldn't reproduce your issue. I think it seems rather unlikely that only surround speakers would be affected and judging by the lack of comments no-one else seems to have this issue.

I think closing this for now is the best idea, we can always reopen if someone can reproduce the issue. What do the others think?

**Comment by MrStahlfelge** [comment]
Yep, we should close it.
